ci/laravel.yml

```diff
@@ -0,0 +1,26 @@
+name: Laravel CI
+on:
+  push:
+    branches:
+    - master
+    - features/*
```

Comment:
As per the guidelines here #156 let's remove any specific branches filtering for a push. Users will be able to add that if they want to.

Reply:
I'm using on: [push] now

https://github.com/actions/starter-workflows/pull/138/commits/9f247c9a93ece1ceb61bfaae2c1b2b1e273560fa